lms/static/js/spec/student_account/access_spec.js

```diff
@@ -51,22 +49,24 @@ define([
             var ajaxSpyAndInitialize = function(that, mode, nextUrl, finishAuthUrl) {
                 // Spy on AJAX requests
                 requests = AjaxHelpers.requests(that);
-
-                // Initialize the access view
-                view = new AccessView({
-                    mode: mode,
-                    thirdPartyAuth: {
+                var options = {
+                    initial_mode: mode,
+                    third_party_auth: {
                         currentProvider: null,
                         providers: [],
                         secondaryProviders: [{name: "provider"}],
                         finishAuthUrl: finishAuthUrl
                     },
-                    nextUrl: nextUrl, // undefined for default
-                    platformName: 'edX',
-                    loginFormDesc: FORM_DESCRIPTION,
-                    registrationFormDesc: FORM_DESCRIPTION,
-                    passwordResetFormDesc: FORM_DESCRIPTION
-                });
+                    login_redirect_url: nextUrl, // undefined for default
+                    platform_name: 'edX',
+                    login_form_desc: FORM_DESCRIPTION,
+                    registration_form_desc: FORM_DESCRIPTION,
+                    password_reset_form_desc: FORM_DESCRIPTION
+                };
+                var logistrationElement = $('#login-and-registration-container');
```

Comment:
There only needs to to be 1 var used at the top of the function, comma separated to define multiple variables.